website_sale_brand_catalogs/controllers/portal.py

```diff
@@ -0,0 +1,32 @@
+from odoo import http
+from odoo.http import request
+
+
+class WebsiteCatalogs(http.Controller):
+    @http.route(
+        [
+            "/catalogs",
+            "/catalogs/page/<int:page>",
+            '/catalogs/brand/<model("product.brand"):brand>',
+            '/catalogs/brand/<model("product.brand"):brand>/page/<int:page>',
+        ],
+        type="http",
+        auth="public",
+        website=True,
+    )
+    def brand_catalogs(self, brand=None, **kw):
+        domain = [("catalogs", "!=", False)]
+        catalog_brand_list = request.env["product.brand"].search(domain)
+
+        values = {}
```

Comment:
The problem is in the get brands when I don't receive a specific one, there I must show all of them.

Reply:
I mean, it doesn't make sense something like:

```
values = {"is_important: True}
if condition:
    values = {"other_values": True}
    return values
```

Because values are overwritten inside the conditional and returned, so if the condition is meth, first values are actually never used.